### PR TITLE
Gift Entry: Fix for allowing post messages from alternate VF origins

### PR DESCRIPTION
**Pull request by voduyemi**
Fix to allow for alternate visualforce domain origins for the elevate payments widget in Gift Entry 

WI: [W-8018691](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07B0000008UmqNIAS/view)

# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js [hunk 1]
```diff
@@ -102,15 +113,16 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     * make sure we're only listening for messages from the correct source in the
     * registerPostMessageListener method.
     */
-    buildVisualforceOriginUrl(domain) {
+    buildVisualforceOriginUrls(domain) {
         let url = `https://${domain}--c.visualforce.com`;
-
-        const currentNamespace = getNamespace(DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName);
+        let alternateUrl = `https://${domain}--c.visual.force.com`;
```
Comment (ptewson-sfdo) [inline src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js:118]: I believe we need to include the 'pod' information in here as well (or else modify the checking logic somehow) - the alternate format looks like this:

${domain}--npsp.cs18.visual.force.com

And I believe we need to grab the  "cs18" section from somewhere - it's an additional piece of information that isn't in the shortened format. I think there are examples of grabbing it like so: 

String pod = [SELECT InstanceName FROM Organization].InstanceName.toLowerCase();

Which appears to work!

File: src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js [hunk 2]
```diff
@@ -42,7 +53,7 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     async connectedCallback() {
         this.PAYMENT_TRANSACTION_STATUS_ENUM = Object.freeze(JSON.parse(await getPaymentTransactionStatusValues()));
         this.domain = await getOrgDomain();
-        this.visualforceOrigin = this.buildVisualforceOriginUrl(this.domain);
+        this._visualforceOrigins = this.buildVisualforceOriginUrls(this.domain);
```
Comment (bdvorachek) [inline src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js:56]: One cleanup thing, `buildVisualforceOriginUrls` can access `this.domain` directly in the body of the function, no need to pass it as an argument.

Comment (voduyemi) [reply src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js]: Done!

File: src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js (hunk 1 above)
Comment (voduyemi) [reply src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js]: Done!

**Pull request merged**